hoomd/pytest/test_filter.py

```diff
@@ -1,14 +1,12 @@
 import pytest
-from hoomd.filter.type_ import Type
-from hoomd.filter.tags import Tags
-from hoomd.filter.set_ import SetDifference, Union, Intersection
-from hoomd.filter.all_ import All
-from hoomd.filter import Null
+from hoomd.filter import (
+    Type, Tags, SetDifference, Union, Intersection, All, Null, CustomFilter)
```

Comment:
I was surprised that yapf isn't automatically alphabetizing this (or using some other obvious ordering). @joaander it looks like neither yapf nor flake8 is run as part of CI (although pydocstyle is), is this an intentional decision because there's a lot of unformatted older code still present? If not, I would definitely recommend adding a test step that fails if yapf makes changes or flake8 raises any errors, that would make your job much easier so you don't have to worry about running these yourself before every release etc.

Reply:
The majority of the codebase has not yet been run through yapf, clang-format, or updated for flake8 compliance . `pydocstyle` runs on a handful of files that developers have added to a specific list.

I don't want to waste time formatting old v2 code only for it to be removed/rewritten for v3. Also, we have a number of outstanding massive PRs. When these are resolved we will add the formatters to CI and enforce formatting going forward but for now we are still in transition.